app/models/squix/print_job_wrapper.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Squix
+  # Squix::PrintJobWrapper
+  class PrintJobWrapper
+    include ActiveModel::Model
+
+    attr_accessor :printer_name, :label_template_name, :label_template_id, :labels, :copies, :errors
+
+    # validate
```

Comment:
So this will be a case of validating on the child print jobs and cascading errors.